addons/uis.md

```diff
@@ -21,6 +23,13 @@ title: User Interfaces
 | [iOS App]({{docu}}/addons/uis/apps/ios.html) | The native iOS app to access openHAB on the go. |
 | [Windows 10 App]({{docu}}/addons/uis/apps/windows.html) | The native Windows 10 app to access openHAB on the go. |
 
+## Iconsets
+
+Basic UI, Classic UI and all apps present a graphical user interface as defined in [Sitemaps]({{base}}/configuration/sitemaps.html) by the help of [Items]({{base}}/configuration/items.html).
+Each Item respectively each Sitemap element may be assoziated with an icon visible next to it.
```

Comment:
I do not understand what you mean by this sentence - consider rewording it.
Consider changing "assoziated" with "associated".